### PR TITLE
refact some code and more test.

### PR DESCRIPTION
**Pull request by tiannian**

### CODE REVIEW

**Review by malefooo — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -31,7 +29,7 @@ async function main () {
 
     actor_bin_instance = (await WebAssembly.instantiate(actor_bin_code, import_object)).instance;
```
Comment (Baoyuantop) [inline index.js:31]: Synced to #3

**Review by Baoyuantop — APPROVED** [review]

**Pull request merged**